bundle/libraries/workspace_path.go

```diff
@@ -36,3 +36,8 @@ func IsWorkspaceLibrary(library *compute.Library) bool {
 
 	return IsWorkspacePath(path)
 }
+
+// IsVolumesPath returns true if the specified path indicates that
```

Comment:
Unfinished comment.

This also conflicts with the other PR, which one do you want to merge first?

Reply:
I'd merge Validator PR first and then would reuse the code from there